.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "build"]
 	path = build
-	url = https://github.com/crossplane/build
+	url = https://github.com/riccap/build
+	branch = 33-fix-xpkg-parameters
```

Comment:
Fixes https://github.com/crossplane/build/issues/33, if you want I can create a PR for that in the upstream repo